global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "5.0.301"
+        "version": "5.0.203"
```

Comment:
This will need to be revered before merging. I don't know why it isn't working on our machines, but other people seem fine :)

Reply:
Got it! Forgot to changed it, now it's solved.